src/Gateway/PayplugResponse.php

```diff
@@ -178,7 +191,7 @@ private function gateway_name($gateway_id){
 		if (str_starts_with($gateway_id, 'oney_')) {
 			return str_replace("_", " ", $gateway_name);
 		}
-		return $gateway_name;
+		return ucwords(str_replace("_", " ", $gateway_name));
```

Comment:
You have this above!
Should add another if! and place on return

You should just have apple_

Reply:
you'll change the behaviour for everyone who's not oney this way!

Reply:
I have fix this method 
just the last line do the job for every payment method